rs/ic_os/network/src/mac_address.rs

```diff
@@ -131,7 +133,17 @@ pub fn generate_mac_address(
         );
         Ok(mgmt_mac)
     } else {
-        let ipmitool_output = get_command_stdout("ipmitool", ["lan", "print"])?;
+        // A bug in our version of ipmitool causes it to exit with an error
+        // status, but we have enough output to work with anyway.
+        // https://github.com/ipmitool/ipmitool/issues/388
+
+        // let ipmitool_output = get_command_stdout("ipmitool", ["lan", "print"])?;
```

Comment:
Is there a reason you're leaving the old call to get_command_stdout commented? And I still see get_command_stdout imported, even though it's not used in the file.

Reply:
cargo clippy beat me to it!